### PR TITLE
CNV-25234: Add kubevirt health status popup

### PR DESCRIPTION
**Pull request by pcbailey**
## 📝 Description

This PR changes the way the health status for OpenShift Virtualization is calculated for reporting in the OCP dashboard status card. It also adds a popup that displays the number of alerts by severity level as well as in a donut chart.

## 🎥 Screenshot
Selection_222

### CODE REVIEW

File: console-extensions.json [hunk 1]
```diff
@@ -453,13 +453,16 @@
...
   },
   {
-    "type": "console.dashboards/overview/health/url",
+    "type": "console.dashboards/overview/health/prometheus",
     "properties": {
-      "name": "%plugin__kubevirt-plugin~Virtualization%",
-      "url": "apis/subresources.kubevirt.io/v1/healthz",
+      "title": "%plugin__kubevirt-plugin~OpenShift Virtualization%",
+      "name": "%plugin__kubevirt-plugin~OpenShift Virtualization%",
+      "queries": ["operator_health_metric{name=\"kubevirt-hyperconvereged\"}"],
```
Comment (sradco) [inline console-extensions.json:460]: The metric name is "kubevirt_hyperconverged_operator_health_status" and no label needs to be used:
```
"queries": ["kubevirt_hyperconverged_operator_health_status"],
```

Comment (pcbailey) [reply console-extensions.json]: Thanks! I took the old query we discussed by mistake. I just pushed the fix.

**Comment by sradco** [comment]
PR #2254 was merged. This PR can now use the new`kubevirt_hyperconverged_operator_health_status` metric.

**Review by upalatucci — APPROVED** [review]

**Pull request merged**